app/src/main/java/org/mozilla/fenix/components/metrics/MetricsStorage.kt

```diff
@@ -121,23 +131,26 @@ internal class DefaultMetricsStorage(
         return false
     }.getOrDefault(false)
 
-    private fun Long.hasBeenMoreThanDaySince(): Boolean =
-        System.currentTimeMillis() - this > dayMillis
-
     private fun Long.toCalendar(): Calendar = Calendar.getInstance(Locale.US).also { calendar ->
         calendar.timeInMillis = this
     }
 
+    private fun Long.withinFirstDay() = this < getInstalledTime() + dayMillis
+
     private fun Long.withinFirstWeek() = this < getInstalledTime() + fullWeekMillis
 
+    private fun Long.withinFirstMonth() = this < getInstalledTime() + monthMillis
+
+    private fun Long.duringSecondDay() = this > getInstalledTime() + dayMillis &&
+        this < getInstalledTime() + dayMillis * 2
+
     private fun Calendar.createNextDay() = (this.clone() as Calendar).also { calendar ->
         calendar.add(Calendar.DAY_OF_MONTH, 1)
     }
 
     companion object {
         private const val dayMillis: Long = 1000 * 60 * 60 * 24
-        private const val windowStartMillis: Long = dayMillis * 2
-        private const val windowEndMillis: Long = dayMillis * 28
+        private const val monthMillis: Long = dayMillis * 28
```

Comment:
Personally preferred `windowEndMillis` to `monthMillis` as months vary in duration - perhaps something like `shortestMonthMillis` ?

Reply:
`windowEndMillis` made better sense with the context of a `windowStart` as well, IMO, but I like your suggestion for more detail in the name